tasks/Task.cpp

```diff
@@ -12,44 +12,89 @@
 #include <iostream>
 #include <json/json.h>
 
+#include <list>
+
 using namespace controldev_websocket;
 using namespace controldev;
 using namespace seasocks;
 
-struct controldev_websocket::JoystickHandler : WebSocket::Handler {
+struct controldev_websocket::Client {
     WebSocket *connection = nullptr;
+    std::string id = "";
+    Json::FastWriter fast;
+    bool is_new = true;
+
+    bool isEmpty(){
+        return connection == nullptr;
+    }
+
+    void steal(Client &other){
+        if (other.isEmpty() and !is_new){
+            return;
+        }
```

Comment:
When would this trigger exactly?

Reply:
`is_new` means the client never stolen the connection, so I use this with an empty check to prevent sending a "new" message when going from pending to connection (on handshake). 

I made that because when you send the handshake message the client expects `{ result: true }` message. Thus, without it, the client would fail the handshake.